packages/init/node-async_hooks.js

```diff
@@ -0,0 +1,229 @@
+import { createHook, AsyncResource } from 'async_hooks';
+
+/// <reference path="./node-async_hooks-symbols.d.ts" />
+
+const asyncHooksWellKnownNameFromDescription = {
+  async_id_symbol: 'nodeAsyncHooksAsyncId',
+  trigger_async_id_symbol: 'nodeAsyncHooksTriggerAsyncId',
+  destroyed: 'nodeAsyncHooksDestroyed',
+};
+
+const promiseAsyncHookFallbackStates = new WeakMap();
+
+const setAsyncSymbol = (description, symbol) => {
+  const wellKnownName = asyncHooksWellKnownNameFromDescription[description];
+  if (!wellKnownName) {
+    throw new Error('Unknown symbol');
+  } else if (!Symbol[wellKnownName]) {
+    Symbol[wellKnownName] = symbol;
```

Comment:
I am uncomfortable making these weird symbols into apparent well-known symbols by adding them as statics to the `Symbol` constructor. Let's discuss before we go forward with this.